n64.mk

```diff
@@ -70,7 +70,7 @@ $(BUILD_DIR)/%.o: $(SOURCE_DIR)/%.S
 	@mkdir -p $(dir $@)
 	set -e; if case $(notdir $(basename $@)) in "rsp"*) true;; *) false;; esac; then \
 		echo "    [RSP] $<"; \
-		$(N64_CC) $(N64_ASFLAGS) -nostartfiles -MMD -Wl,-Ttext=0x1000 -Wl,-Tdata=0x0 -Wl,-e0x1000 -o $@ $<; \
+		$(N64_CC) $(ASFLAGS) -nostartfiles -MMD -Wl,-Ttext=0x1000 -Wl,-Tdata=0x0 -Wl,-e0x1000 -o $@ $<; \
```

Comment:
Strange. I had to make this change before because an include path was missing. I just tried rebuilding without the change and it worked again. Maybe a caching issue? Anyway, I'll roll it back.